modules/cnf-topology-aware-lifecycle-operator-precache-feature.adoc

```diff
@@ -0,0 +1,234 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc
+
+:_content-type: PROCEDURE
+[id="talo-precache-feature_{context}"]
+= Using the container image pre-cache feature
+
+The clusters might have limited bandwidth to access the container image registry, which can cause a timeout before the updates are completed. 
+The pre-cache feature allows the required container images to be present on the spoke cluster before the update starts.
+
+[NOTE]
+====
+The time of the update is not set by TALO. You can apply the `ClusterGroupUpgrade` CR at the beginning of the update by manual application or by external automation.
+====
+
+The container image pre-caching starts when the `preCaching` field is set to `true` in the `ClusterGroupUpgrade` CR. After a successful pre-caching process, you can start remediating policies. The remediation actions start when the `enable` field is set to `true`.
+
+[id="talo-precache-autocreated-cgu-for-ztp_{context}"]
+== About the auto-created ClusterGroupUpgrade CR for ZTP
```

Comment:
This file is talking about image precaching, which doesn't look like the right place for ztp-install CGU (doesn't use image precaching). 
If we have a consolidated end-to-end upgrade guide, then it is more suitable to go there, as the policies from ZTP would be reused in upgrade.